### ISSUE TITLE
Command to create the md note

### ISSUE THREAD

**Issue by notuntoward** [daeh/zotero-markdb-connect#144]
With this plugin, there's now a frictionless connection between a zotero item to a corresponding note created by the obsidian [zotero integration plugin](https://github.com/mgmeyers/obsidian-zotero-integration).  That's a big improvement.  But after creating a new zotero item, a few steps are still required to  move over to obsidian and to create the new zotero item's obsidian note.

I'd love it if zotero-markdown-connect had it's own command to create the obsidian note, so that creating a linked obsidian note is as simple as creating a zotero internal note.

Ideally, this new zotero-markdown-connect would have the templating ability needed to export the zotero item's metadata to obsidian (even more ideal would be the _same_ nunjucks templating as the [zotero integration plugin](https://github.com/mgmeyers/obsidian-zotero-integration).  But I'd be pretty happy with a new obsidian note with:
- a filename matching the betterbibtex citekey, 
- a link back to the source zotero item.
- links for any zotero item attachments

**Comment by daeh** [comment]
Hi @notuntoward — Thanks for the suggestion. The original design philosophy of this plugin was to be a lightweight helper for existing lit-note workflows. Conceivably I could look at making it possible to trigger note creation by something like `obsidian-zotero-integration` from within Zotero, but it's unlikely to make sense for this plugin to try to cover note creation.

**Comment by notuntoward** [comment]
Fair enough.  For the zotero actions and tags plugin, I've now written the javascript to write an obsidian note.

Is there a way for this script to trigger MardDB-Connect Sync Tags?  A plain javascript call would be nice, but I'm not sure if this is possible.  But I do know that the better bibtex plugin can be triggered from Javascript via its [JSON_RPC](https://retorque.re/zotero-better-bibtex/exporting/json-rpc/index.html)

**Comment by daeh** [comment]
What OS are you using? On MacOS you could conceivably trigger MDBC sync using JSA to automate the Zotero application menu.

**Comment by notuntoward** [comment]
I'm on Windows.  I'm more of a javascript vibe coder, but to me it looks like this could work.  Is there already an MDBC endpoint for this?

**Comment by finedust** [comment]
>  Conceivably I could look at making it possible to trigger note creation by something like `obsidian-zotero-integration` from within Zotero, but it's unlikely to make sense for this plugin to try to cover note creation.

Of course one option would be to just create the file and add the Zotero's metadata as YAML properties. One could use the selected Obsidian directory for sync, even though this would be slightly inconvenient for me (Zotero's plugin points to a bigger directory than the one I usually write notes to, because few of them are them moved around).

However, maybe one could also directly use the [obsidian citation plugin](https://github.com/hans/obsidian-citation-plugin) that is there for this exact purpose! I'm not familiar with the API of Obsidian, but if Zotero-markdb-connect could call the "open new note" action of the Obsidian's plugin it would literally suffice to pass the citekey and then the user could customize the note template using one single workflow, at basically no added cost for this plugin.
